skonfig/__init__.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import configparser
+import logging
+import os
+import sys
+import uuid
+
+import skonfig.cdist
+import skonfig.dump
+
+
+SKONFIG_CONFIGURATION_DIRECTORY = "{}/.skonfig".format(os.getenv("HOME"))
+SKONFIG_CONFIGURATION_FILE_PATH = "{}/config".format(SKONFIG_CONFIGURATION_DIRECTORY)
+
+
+logger = logging.getLogger("skonfig")
+
+
+def run():
+    if os.path.basename(sys.argv[0])[:2] == "__":
+        return skonfig.cdist.run_emulator()
+    arguments = get_arguments()
+    if arguments.verbose > 1:
```

Comment:
Do we need this `if`? Isn't the logging configured by the old cdist core?

Reply:
It is not yet, when this function is called, but we want to write some DEBUG log before cdist arrives to the party, and also display it.

It's kind of hack because I didn't came up with quick nice solution. Definetly room for improvement, but I rather leave it for the future.